pkg/server/cluster_server.go

```diff
@@ -27,6 +27,7 @@ const (
 	caBundleFilePath    = "/etc/kubernetes/kubelet-ca.crt"
 	cloudProviderCAPath = "/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem"
 	additionalCAPath    = "/etc/pki/ca-trust/source/anchors/openshift-config-user-ca-bundle.crt"
+	kubeConfigFilePAth  = "/etc/kubernetes/kubeconfig"
```

Comment:
we technically have a `defaultMachineKubeConfPath` already

Also, should probably remove the part where that gets appended to the in-cluster server from the bootstrap manifest (there's a appendKubeConfig func) or we'l double write it I think?

Reply:
I think I should remove the `appendKubeConfig` func instead right? Because that func just writes the old bootstrap data and doesn't take into account if it updated.